docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+# Cellxgene Curation Tutorial
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png) --->
+
+<img align="right" src="https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png">
+
+
+<!--- ## Overview --->
+<!--- purpose: Motivate submission to the data portal give context --->
+
+[Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) is a system which is optimized for access, exploration and reuse of single cell data. In order to achieve these goals, the cellxgene platform currently accepts curated [AnnData](https://anndata.readthedocs.io/en/latest/#) objects adhering to a [succinct data schema](corpora_schema.md). Adherence to a standardized data schema allows for efficient navigation and integration of the growing number of single cell datasets that are becoming available. In this tutorial, you will learn the essential information for curating a single cell dataset using CZI's curation tools and uploading to the data portal. Hosting your data on the cellxgene portal will offer the following benefits:
+ - link permanence (you can reference in your publication without ever worrying about dead links)
+ - sharing of private datasets with collaborators (keep the data private until it is ready for publication)
+ - no barrier for readers to explore your dataset (and no need for you to build your own single cell data explorer)
+ - accesibility of your dataset in the major single cell data formats (including `AnnData`, `seurat`, and `loom`)
```

Comment:
__accesibility -> accessibility__
Recommend spell-checking.

I might also prefer "availability" since "accessibility" has other connotations. 

Free data format conversion. Data consumers can download your dataset in the the major single cell data  formats ...

Reply:
changed the typo

was the last line in this comment a suggestion for phrasing or are you saying that I should that data conversion is free?